samples/graph-activity-feed-broadcast/csharp/ActivityFeedBroadcast/wwwroot/js/auth.js

```diff
@@ -38,29 +43,33 @@ function getToken() {
         microsoftTeams.authentication.authenticate({
             url: window.location.origin + "/Auth/Start",
             width: 600,
-            height: 535
-        }).then(result => {
-            resolve(result);
-        }).catch(reason => {
-            reject(reason);
+            height: 535,
+            successCallback: result => {
+                resolve(result);
+            },
+            failureCallback: reason => {
+                reject(reason);
+            }
         });
     });
 }
 
 function getClientSideToken() {
-
     return new Promise((resolve, reject) => {
-        microsoftTeams.authentication.getAuthToken().then(result => {
-            resolve(result);
-        }).catch(error => {
-            reject("Error getting token: " + error);
+        microsoftTeams.authentication.getAuthToken({
+            successCallback: (result) => {
+                resolve(result);
+            },
+            failureCallback: function (error) {
+                reject("Error getting token: " + error);
+            }
         });
     });
 }
 
 function getServerSideToken(clientSideToken) {
     return new Promise((resolve, reject) => {
-        microsoftTeams.app.getContext().then((context) => {
+        microsoftTeams.getContext((context) => {
```

Comment:
Revert it to use :  microsoftTeams.app.getContext().then((context) => {

Reply:
changed